docs/source/_src_getting_started/submodules.rst

```diff
@@ -0,0 +1,43 @@
+====================
+
+The top-level submodules of tuduatpy are:
+
+* `tudatpy.numerical_simulation <https://py.api.tudat.space/en/latest/astro.html>`_ This submodule contains the interfaces for the primary application of Tudatpy: numerical state propagation and estimation. The functionality in this submodule consists of a large number of interconnected elements that work together as a whole. The usability of separate functions/classes in this submodule *outside* of the Tudatpy framework is very limited, and this functionaluty is typically only used within a Tudatpy numerical simulation. Its further subdivision into submodules is discussed in more detail below.
+* `tudatpy.astro <https://py.api.tudat.space/en/latest/astro.html>`_ This submodule contains various (semi-)standalone functions for astrodynamics applications, which can be used very well outside of a Tudat application/propagation. Submodules contain lists of frame conversions, element conversion, elementary orbit calculations, *etc.*.
+* `tudatpy.trajectory_design <https://py.api.tudat.space/en/latest/trajectory_design.html>`_ This submodule contains functionality for the preliminary design of a full (transfer) orbit, using for instance a Muliple Gravity Assist (MGA) or a low-thrust system. It relies of functionality in the ``astro`` submodule. It is largely independent of the ``numerical_simulation`` submodule, but does contain interface functitons to allow the preliminary design to be used as an initial guess for a full numerical propagation.
+* `tudatpy.math <https://py.api.tudat.space/en/latest/math.html>`_  This submodule contains various functions and classes for purely mathematical operations, such as numerical interpolation, numerical integration, *etc.*.
+* `tudatpy.interface <https://py.api.tudat.space/en/latest/interface.html>`_ This submodule contains functionality to interface with various external packages which Tudat uses, such as `Spice <https://naif.jpl.nasa.gov/naif/toolkit.html>`_
+* `tudatpy.plotting <https://py.api.tudat.space/en/latest/plotting.html>`_ This submodule contains various pieces of functionality to support the easy plotting of results generated with Tudatpy. Unlike most of the main Tudatpy submodules (which are written in C++, and exposed to Python), this submodule is written largely in Python
+* `tudatpy.util <https://py.api.tudat.space/en/latest/util.html>`_ This submodule contains various small pieces of functionality to support the easy post-processing of results generated with Tudatpy. Unlike most of the main Tudatpy submodules (which are written in C++, and exposed to Python), this submodule is written largely in Python
+* `tudatpy.io <https://py.api.tudat.space/en/latest/io.html>`_ This submodule contains various pieces of functionality for file input-output in Tudatpy. Unlike most of the main Tudatpy submodules (which are written in C++, and exposed to Python), this submodule is written largely in Python
+
+The numerical_simulation submodules
```

Comment:
I guess the idea is to have a dedicated section focusing on each top-level submodules? similar to what you now have for numerical_simulation?